src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -141,6 +142,13 @@ public void parseCommand_view() throws Exception {
         assertTrue(parser.parseCommand(ViewCommand.COMMAND_WORD + " c" + " 1") instanceof ViewCommand);
     }
 
+    @Test
+    public void parseCommand_sort() throws Exception {
+        assertTrue(parser.parseCommand(SortCommand.COMMAND_WORD + " c") instanceof SortCommand);
+        assertTrue(parser.parseCommand(SortCommand.COMMAND_WORD + " c"
+            + " start/14-05-2012 10:51")instanceof SortCommand);
```

Comment:
Just a nit here. A space is missing between `+ " start/14-05-2012 10:51")` and `instanceof SortCommand);`.